lib/mongo/client.rb

```diff
@@ -53,6 +54,7 @@ class Client
       :auth_mech,
       :auth_mech_properties,
       :auth_source,
+      :auto_encryption_opts,
```

Comment:
We generally only use "opts" when "options" is already taken. For example methods take `opts` as argument if `options` is an attribute on the class.

Reply:
Good to know! Thanks.